assignments/prototypes.js

```diff
@@ -16,12 +16,49 @@
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
 
-/*
-  === CharacterStats ===
-  * healthPoints
-  * takeDamage() // prototype method -> returns the string '<object name> took damage.'
-  * should inherit destroy() from GameObject's prototype
-*/
+function GameObject(attr) {
+  this.createdAt = attr.createdAt;
+  this.name = attr.name;
+  this.dimensions = attr.dimensions;
+}
+
+GameObject.prototype.destroy = function () {
+  return `${this.name} was removed from the game`;
+}
+
+let game1 = new GameObject({
+  createdAt: new Date(),
+  name: `Game1`,
+  dimensions: {
+    lenght: 1,
+    width: 2,
+    height: 3,
+  }
+});
+
+console.log(game1);
+console.log(game1.createdAt);
+console.log(game1.name);
+console.log(game1.dimensions);
+console.log(game1.destroy());
+
+
+/* *  CharacterStats ===
+ * healthPoints
+ * takeDamage() // prototype method -> returns the string '<object name> took damage.'
+ * should inherit destroy() from GameObject's prototype
+ */
+
+function CharacterStats(theGoods) {
+  this.healthPoints = healthPoints;
+  GameObject.prototype.destroy = function () {
+
+
+  }
+}
+
```

Comment:
You're on the right track. Keep going. Remember to call up to your parent constructor to get their attributes and to use Object.create to get the parent's methods.